tutorial/creating_packages/preparing_the_build.rst

```diff
@@ -93,7 +94,8 @@ As you can see:
 * Based on the value of the ``with_fmt`` option:
 
     - We install or not the ``fmt/8.1.1`` Conan package.
-    - We require or not a minimum C++ standard as the *fmt* library requires at least C++11.
+    - We require or not a minimum and a maximum C++ standard as the *fmt* library requires at least C++11 and our
```

Comment:
Some clarification might be needed here: it is not that "our settings support at most C++14", as that could mean "our Conan settings like ``compiler.version=xxx`` compiler does not support C++17. This kind of check shouldn't be done in recipes.
There is a new ``profile_plugin.py`` plugin that can be used to test and check the validity of our settings, and it could be used to raise if some compiler doesn't support some C++ standard (I think the default implementation of the plugin already does that?)

So maybe this sentence should be clarified: "and our build system or source code would break if tried to compile with a C++ standard higher than 14"